packages/frontend2/src/app/_components/table/basic-table.tsx

```diff
@@ -44,6 +65,7 @@ export function BasicTable<T extends BasicEntry>({
                 colSpan={header.colSpan}
                 className={header.column.columnDef.meta?.headClassName}
                 tooltip={header.column.columnDef.meta?.tooltip}
+                style={{ ...getCommonPinningStyles(header.column) }}
```

Comment:
why spreading if you can simply do this?
```suggestion
                style={getCommonPinningStyles(header.column)}
```